Sources/VenmoPayments/VenmoCheckoutRequest.swift

```diff
@@ -0,0 +1,14 @@
+import Foundation
+
+/// Used to configure options for approving a Venmo Checkout order
+public struct VenmoCheckoutRequest {
+    
+    /// The order ID associated with the request
+    public let orderID: String
+    
+    /// Creates an instance of VenmoCheckoutRequest
+    ///  - Parameter orderID: The ID of an order to be approved
+    public init(orderID: String) {
+        self.orderID = orderID
+    }
```

Comment:
```suggestion
    /// Creates an instance of VenmoCheckoutRequest
    ///  - Parameter orderID: The ID of an order to be approved
    public init(orderID: String) {
        self.orderID = orderID
    }
```
small nitpick: Can we leverage the default constructor here like in `VenmoCheckoutResult`?